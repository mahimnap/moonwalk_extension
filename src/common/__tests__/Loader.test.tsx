import React from "react";

import TestRenderer from "../../helpers/testRenderer";
import Loader from "../Loader";

describe("Loader", () => {
  it("renders correctly", () => {
    const tree = TestRenderer(<Loader />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
