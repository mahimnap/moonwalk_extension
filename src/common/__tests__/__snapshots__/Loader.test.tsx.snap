// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
      },
    ]
  }
>
  <ActivityIndicator
    animating={true}
    color="#999999"
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
